src/screens/ProfileScreen/elements/KarmaLock.tsx

```diff
@@ -45,26 +45,28 @@ export const KarmaLock = (props: {onPressCreatePost: () => void}) => {
           contentStyle={{
             borderRadius: 10,
             padding: 16,
-            height: 'auto'
+            height: 'auto',
+            backgroundColor: COLORS.almostBlack
           }}
           content={
             <View>
               <Text
                 style={{
-                  fontWeight: '600',
+                  fontFamily: fonts.inter[600],
                   fontSize: normalize(16),
                   lineHeight: normalize(24),
-                  marginBottom: normalize(8)
+                  marginBottom: normalize(8),
+                  color: COLORS.white2
                 }}>
                 What is my Karma Score?
               </Text>
               <Text
                 style={{
-                  fontWeight: '400',
+                  fontFamily: fonts.inter[400],
                   fontSize: normalize(12),
                   lineHeight: normalize(18),
                   marginBottom: normalize(8),
-                  color: '#69707C'
+                  color: COLORS.gray500
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `fontFamily` property has been added to the style objects for the Text components, and the `color` property has been changed. This improves consistency in typography and color scheme across the application.
```diff
- fontWeight: '600',
+ fontFamily: fonts.inter[600],
- color: '#69707C'
+ color: COLORS.gray500
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->